jujugui/static/gui/src/app/app.js

```diff
@@ -1527,9 +1527,7 @@ YUI.add('juju-gui', function(Y) {
     _renderProviderLogo: function() {
       const container = document.getElementById('provider-logo-container');
       const cloudProvider = this.env.get('providerType');
-      const providerDetails = cloudProvider ?
-        views.utils.getCloudProviderDetails(cloudProvider) :
-        {};
+      let providerDetails = views.utils.getCloudProviderDetails(cloudProvider);
```

Comment:
Yeah it's safe, `getCloudProviderDetails` returns undefined when there is no provider found. We need this component to get re-rendered when values change so that the css animations work as expected so we can't simply `unmount` as I was hoping to do if none was provided. This is why the `providerDetails` value gets set to an empty object below.